packages/sanes-browser-extension/config/manifest-production.json

```diff
@@ -19,7 +19,7 @@
   },
   "content_scripts": [],
   "externally_connectable": {
-    "matches": ["http://localhost/*", "https://*.iov.one/*"]
+    "matches": ["http://localhost/*", "https://*.neuma.io/*"]
```

Comment:
Removing `https://*.iov.one/*` from this whitelist disallows `governance.iov.one` to send messages to the browser extension, breaking the governance app